async_service/trio.py

```diff
@@ -234,6 +232,9 @@ def _track_current_task(
     ) -> None:
         self.logger.debug("running task '%s[daemon=%s]'", name, daemon)
         done, cancel_scope = self._track_current_task(parent)
+        if parent not in self._service_task_dag:
+            self.logger.debug("New root task %s[daemon=%s] added to DAG", name, daemon)
```

Comment:
Wouldn't it be clearer to have this as the `else` block of that `if` in line 206?

Reply:
That's where I put it originally but I don't have access to the task name or the daemon flag there and I like trying to keep parity between the logging messages the two managers emit.